persistence/sql/persister_courier.go

```diff
@@ -26,10 +26,17 @@ func (p *Persister) AddMessage(ctx context.Context, m *courier.Message) error {
 func (p *Persister) NextMessages(ctx context.Context, limit uint8) ([]courier.Message, error) {
 	var m []courier.Message
 	if err := p.Transaction(ctx, func(ctx context.Context, tx *pop.Connection) error {
+		query := "SELECT * FROM %s WHERE status = ? ORDER BY created_at ASC LIMIT ?"
```

Comment:
This makes sense to me! One question would be if you could add a test for this. My proposal is:

1. Create 50 messages
2. Run 50 `NextMessages` in goroutines with limit of 1
3. Expect to receive 50 different messages

Reply:
Yeah will do!